c/lang/security/insecure-use-string-copy-fn.yaml

```diff
@@ -1,19 +1,17 @@
 rules:
   - id: insecure-use-string-copy-fn
-    pattern-either:
-      - pattern: strcpy(...)
-      - pattern: strncpy(...)
+    patterns:
+      - pattern: $X(...)
+      - metavariable-regex:
```

Comment:
I suspect this is slow compared to the prior implementation.

Reply:
@inkz If we take this approach, is this a simple autofix for us to add?

Reply:
@aryx Got any ideas for how I could approach this for both autofix + using metavariables in the rule message + being performant?

Reply:
@dlukeomalley it is easy ti change func name https://semgrep.dev/s/XZ1A
but I am not 100% sure that arguments will be correct.
Also there is nice comment/pr from one of our users on this rule: https://github.com/returntocorp/semgrep-rules/pull/1824

Reply:
I removed this change for now.